charts/prometheus-pushgateway/values.yaml

```diff
@@ -126,6 +126,14 @@ securityContext:
   runAsUser: 65534
   runAsNonRoot: true
 
+## Security context to be added to push-gateway containers
+## Having a separate variable as securityContext differs for pods and containers.
+containerSecurityContext:
+  allowPrivilegeEscalation: false
```

Comment:
imho these vlaues should not be added by default or updating the chart for some users might break.